frontend/app/src/components/history/LocationDisplay.vue

```diff
@@ -23,18 +23,22 @@ const { identifier, detailPath } = toRefs(props);
 const { locationData } = useLocations();
 const location = locationData(identifier);
 
+const name = computed(() => get(location)?.name ?? '')
```

Comment:
@kelsos

Reply:
Yes I saw the linting failure :)

We need some pre-commit / lint-staged support. I tried that last week but it is a bit more complicated with our setup here.